libmscore/glissando.cpp

```diff
@@ -370,6 +380,21 @@ void Glissando::write(Xml& xml) const
       if (_showText && !_text.isEmpty())
             xml.tag("text", _text);
       xml.tag("subtype", int(_glissandoType));
+      writeProperty(xml, P_ID::PLAY_GLISSANDO);
+      switch ( glissandoStyle()) {
+          case MScore::GlissandoStyle::BLACK_KEYS:
```

Comment:
Well in fact is not a very good idea. A good solution for storing strings instead of enum casted to int would be to create a Property Type like it's done for Direction. Check https://github.com/musescore/MuseScore/blob/master/libmscore/xml.cpp#L394 to see how the property types are handled